common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -34,4 +34,14 @@ trait ABTestSwitches {
     exposeClientSide = true,
   )
 
+  Switch(
+    ABTests,
+    "ab-scroll-depth",
+    "Send scroll depth tracking data",
+    owners = Seq(Owner.withName("dotcom.platform")),
```

Comment:
Semantically I'm not sure this makes sense, as I think this is meant to be a person's name, but the only place I think this is used is in the [ExpiringSwitchesEmailJob](https://github.com/guardian/frontend/blob/main/admin/app/jobs/ExpiringSwitchesEmailJob.scala#L25-L26) - we could use our dotcom email?

Reply:
I went with the `withEmail` pattern